frontend/packages/dev-console/src/components/monitoring/queries.ts

```diff
@@ -71,12 +71,21 @@ export const monitoringDashboardQueries: MonitoringQuery[] = [
     humanize: humanizeBinaryBytes,
     byteDataType: ByteDataTypes.BinaryBytes,
   },
+  {
+    query: _.template(
+      `topk(25, sort_desc(sum((kubelet_volume_stats_used_bytes {namespace='<%= namespace %>'}* on (namespace,persistentvolumeclaim) group_right() kube_pod_spec_volumes_persistentvolumeclaims_info)) by (namespace,pod)))`,
+    ),
+    chartType: GraphTypes.line,
+    title: 'Local Storage Usage',
+    humanize: humanizeBinaryBytes,
+    byteDataType: ByteDataTypes.BinaryBytes,
+  },
   {
     query: _.template(
       `topk(25, sort_desc(sum(pod:container_fs_usage_bytes:sum{container="",pod!="",namespace='<%= namespace %>'}) BY (pod, namespace)))`,
     ),
     chartType: GraphTypes.line,
-    title: 'Filesystem Usage',
+    title: 'Attached Storage Usage',
```

Comment:
I believe the metric used in the query refers to the amount of space the containers rootfs uses, which I don't think is particularly interesting to have on display here.

Reply:
@serenamarie125 do you still want to keep it or shall I remove?